repos/system_upgrade/el7toel8/actors/migratebrltty/actor.py

```diff
@@ -0,0 +1,35 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import BrlttyMigrationDecision
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_generic
+
+
+class MigrateBrltty(Actor):
+    """
+    Migrate brltty configuration files.
+    """
+
+    name = 'migrate_brltty'
+    consumes = (BrlttyMigrationDecision,)
+    produces = (Report,)
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for decision in self.consume(BrlttyMigrationDecision):
+            report_summary = ''
```

Comment:
Same comment about report_summary = None and string concatenation as above.

Reply:
Fixed